ctpolicy/applepolicy.go

```diff
@@ -41,6 +42,25 @@ func (appleP AppleCTPolicy) LogsByGroup(cert *x509.Certificate, approved *loglis
 	return groups, err
 }
 
+// LogsByGroup2 describes submission requirements for embedded SCTs according to
+// https://support.apple.com/en-us/HT205280. Returns data even when error emitted.
+func (appleP AppleCTPolicy) LogsByGroup2(cert *x509.Certificate, approved *loglist2.LogList) (LogPolicyData, error) {
```

Comment:
Is anyone using (or expected to use) this code with LogListv1 data? If not, maybe take the opportunity and just convert it to LLv2 and drop support for LLV1 entirely (or is there a reason not to do that?)

Reply:
This PR is preparation: creating code for LLv2 and testing it. The next one is expected to be the switch and deletion/removal of LLv1.

Reply:
Aha, cool :)